### PR TITLE
Python: Top 10 libraries used in python

### LINKED ISSUE #4797: Python: Top 10 libraries used in python
### Description

Document on top 10 libraries used in python
- Approach: Introduction, properly commented code, images wherever required and applications.
- [x] GWOC21
-Under hacktoberfest-accepted label as well

### Domain

Python_Domain

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Anujakumari**
<hr>

## Description 📜

In this Video i have shown Top 10 libraries used in python. 
Fixes #4797 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [x] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Anujakumari** [comment]
@apeksha235, 
plz review my PR.
Thanks!

File: Python/Libraries/Top_10_Python_Libraries_Video.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Top_10_Python_Libraries_Video
+
```
Comment (apeksha235) [inline Python/Libraries/Top_10_Python_Libraries_Video.md:2]: Just mention the names of the libraries that you've explained in the video

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@Anujakumari

File: Python/Libraries/Top_10_Python_Libraries_Video.md (hunk 1 above)
Comment (Anujakumari) [reply Python/Libraries/Top_10_Python_Libraries_Video.md:2]: In Description? @apeksha235

Comment (Anujakumari) [reply Python/Libraries/Top_10_Python_Libraries_Video.md:13]: @apeksha235 
Done! Now is it ok?

Comment (apeksha235) [reply Python/Libraries/Top_10_Python_Libraries_Video.md:13]: @Anujakumari Also mention that you've explained these in the video

Comment (Anujakumari) [reply Python/Libraries/Top_10_Python_Libraries_Video.md:2]: @apeksha235 , Now is it ok?

Comment (Anujakumari) [reply Python/Libraries/Top_10_Python_Libraries_Video.md:2]: @apeksha235 , plz check once.

**Review by kumarankm — APPROVED** [review]
@adityabisoi I think we can merge this

**Pull request merged**